setup.py

```diff
@@ -224,6 +224,30 @@ def run(self):
         except OSError as e:
             print("Error deleting %s - %s." % (e.filename, e.strerror))
 
+class merge_license_files():
```

Comment:
It doesn't seem like this is actually being used anywhere.

Reply:
Should we make a class like that https://github.com/numpy/numpy/blob/29cb0ef6bb47a942d77bf7479a870a0c88af6fc7/setup.py#L178 
to concatenate all license files into a single file.
Or we put it into https://github.com/sympy/sympy/blob/master/setup.py#L197